packages/frontend-2/components/workspace/JoinPage.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <HeaderWithEmptyPage empty-header>
```

Comment:
I changed this in my branch, but I'm changing this `HeaderWithEmptyPage` to be a layout and a header component, so we dont need to wrap the whole page in it